app/src/main/res/values/strings.xml

```diff
@@ -114,6 +114,9 @@
     <string name="off_no_salary_from">Off no salary from</string>
     <string name="off_no_salary_to">Off no salary to</string>
     <string name="off_have_salary_insurance_coverage">Off have salary - Insurance coverage</string>
+    <string name="edit">Edit</string>
+    <string name="delete">Delete</string>
+    <string name="being_handled_by">Being handled by</string>
```

Comment:
FATAL: [MissingTranslation] "`being_handled_by`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)